docs/contributing/style-guide.md

```diff
@@ -167,171 +146,6 @@ impl<OnChange:Callback0> HierarchicalTransform<OnChange> {
 ```
 
 
-### Vertical spacing
-We use the following amount of vertical spacing:
-- 3 blank lines after imports.
-- 3 blank lines before each section.
-- 2 blank lines before each sub-section.
-- 1 blank line after each section / sub-section.
-- 1 blank line before functions / structures / impls.
-- 1 blank line at the end of the file.
-
-Please note that vertical spacing overlaps. For example, if there is a section
-after imports, the total number of blank lines is 3, not 6.
-
-
-### Multiline Expressions
-Most (preferably all) expressions should be single line. Multiline expressions
-are hard to read and introduce noise in the code. Often, it is also an indicator
-of code that is not properly refactored. Try to refactor parts of multiline
-expressions to well-named variables, and divide them to several single-line
-expressions.
```

Comment:
We should consider keeping the requirement of "Try to refactor parts of multiline
expressions to well-named variables", however I'm not very fond of hard "one-line" restriction.